tests/console/flask.pm

```diff
@@ -44,11 +44,14 @@ sub run {
         assert_script_run 'kill $!';
     }
 
-    zypper_call "in uwsgi uwsgi-python3";
-    record_info 'uwsgi';
-    assert_script_run '/usr/sbin/uwsgi --plugin python3 --http-socket :7000 --mount /=flask_app:app & sleep 10';
-    assert_script_run 'curl -s http://localhost:7000/ | grep "Hello World"';
-    assert_script_run 'kill $!';
+    # uwsgi is not available on SLE
+    unless (is_sle) {
+        zypper_call "in uwsgi uwsgi-python3";
+        record_info 'uwsgi';
+        assert_script_run '/usr/sbin/uwsgi --plugin python3 --http-socket :7000 --mount /=flask_app:app & sleep 10';
+        assert_script_run 'curl -s http://localhost:7000/ | grep "Hello World"';
+        assert_script_run 'kill $!';
```

Comment:
This is unrelated change, please move it to a separate commit

Reply:
I will do this after I get some other reviews so I don't need to do git magic every time I want to change something in non-last commit.

Reply:
Cool then :D

Reply:
Done.